packages/contracts-bedrock/deploy-config/base-sepolia.json

```diff
@@ -0,0 +1,62 @@
+{
+  "finalSystemOwner": "0x608081689Fe46936fB2fBDF7552CbB1D80ad4822",
+  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
```

Comment:
I think this is causing the failure:

```suggestion
  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
```

Reply:
Committing this to see.

Reply:
I am unclear what the diff is here

Reply:
Ah ok github bug, the fix is already merged

Reply:
deletes that line. The var name is now `superchainConfigGuardian`